src/main/java/util/Tier.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+public enum Tier {
```

Comment:
I need to unify all of the different difficulty data structures in this bot someday

Reply:
That was the eventual thought yes.